### PR TITLE
Bump hdk 0.4.0-dev deps

### PR DESCRIPTION
**Pull request by mattyg**
- Bump packages to latest compatible with 0.4.0-dev
- Tryorama test for updating profile
- fix build error
- bump version to 0.400.0-dev.3

### CODE REVIEW

**Review by matthme — APPROVED** [review]

File: tests/src/profile.test.ts [hunk 1]
```diff
@@ -32,6 +32,26 @@ test("create Profile", async () => {
 
     const profileStatus = get(alice.store.myProfile);
     assert.equal(profileStatus.status, "complete");
-    assert.ok((profileStatus as any).value);
+    assert.equal((profileStatus as any).value.actionHash.toString(), profile.actionHash.toString());
+
+
+    // Alice updates a Profile
```
Comment (mattyg) [inline tests/src/profile.test.ts:38]: test that myProfile subscriber gets triggered on profile update

File: ui/package.json [hunk 2]
```diff
@@ -31,9 +31,9 @@
     "localize:build": "lit-localize build"
   },
   "dependencies": {
-    "@holochain-open-dev/elements": "0.400.0-dev.0",
-    "@holochain-open-dev/stores": "0.400.0-dev.0",
-    "@holochain-open-dev/utils": "0.400.0-dev.0",
+    "@holochain-open-dev/elements": "0.400.0-dev.2",
```
Comment (mattyg) [inline ui/package.json:34]: bump ui deps to latest

File: ui/src/mocks.ts [hunk 3]
```diff
@@ -61,8 +61,9 @@ export class ProfilesZomeMock extends ZomeMock implements AppClient {
     myPubKey?: AgentPubKey,
     installedAppId?: InstalledAppId,
   ) {
-    super("lobby", "profiles", myPubKey);
-    this.installedAppId = installedAppId ? installedAppId : 'dummy-app-id';
+    const appId  = installedAppId ? installedAppId : 'dummy-app-id';
+    super("lobby", "profiles", installedAppId, myPubKey);
```
Comment (mattyg) [inline ui/src/mocks.ts:65]: the super class constructor parameters changed

**Comment by guillemcordoba** [comment]
I'm only maintaining the nixify branch so I'm fine with this!

**Review by guillemcordoba — APPROVED** [review]

**Pull request merged**